packages/api-reference/src/components/Content/Schema/Schema.vue

```diff
@@ -124,16 +128,19 @@ watch(
       <!-- Schema description -->
       <div
         v-if="
-          resolvedSchema?.description &&
-          typeof resolvedSchema.description === 'string' &&
-          !resolvedSchema.allOf &&
-          !resolvedSchema.oneOf &&
-          !resolvedSchema.anyOf &&
+          schema &&
```

Comment:
I'd probably use a helper function here with some docs that explains what its looking for

Reply:
or just a var

Reply:
done in #5782